animations.md

```diff
@@ -1,311 +1,377 @@
+
+### Controlling how animations run
+
+This section briefly describes some of the common APIs you will use for
+creating animation effects and controlling how they run.
+
+#### [AnimationController](http://docs.flutter.io/flutter/material/AnimationController-class.html)
+
+Use this class to specify how the animation should run. The
+<code>[AnimationController](http://docs.flutter.io/flutter/material/AnimationController-class.html)</code>
+class lets you define important characteristics of the animation, such as its
+duration and playback direction (forward or reverse).
+
+The following snippet shows how you might create an animation that runs for 500 milliseconds:
+
+
+<pre class=prettyprint>
+final AnimationController controller =
+    new AnimationController(duration: const Duration(milliseconds: 500));
+</pre>
+
+#### [CurvedAnimation](http://docs.flutter.io/flutter/material/CurvedAnimation-class.html)
+
+Use this class to apply a non-linear curve to an animation object it wraps. By
+applying different curve types, you can visually modify the spatial movement
+pattern for the wrapped animation object. The
+<code>[Curves](http://docs.flutter.io/flutter/animation/Curves-class.html)</code> class lists the
+available curve types, such as
+<code>[bounceIn](http://docs.flutter.io/flutter/animation/Curves/bounceIn-constant.html)</code>,
+<code>[easeOut](http://docs.flutter.io/flutter/animation/Curves/easeOut-constant.html)</code>, and
+others.
+
+The following snippet shows how you might create a curved animation object that
+wraps a linear animation produced by an animation controller and applies the
+<code>[easeIn](http://docs.flutter.io/flutter/animation/Curves/easeIn-constant.html)</code> curve
+type to it.
+
+
+<pre class=prettyprint>
+final AnimationController controller =
+    new AnimationController(duration: const Duration(milliseconds: 500));
+final curve = new CurvedAnimation(
+    parent: controller,
+    curve: Curves.easeIn
+);
+</pre>
+
+#### [Tween](http://docs.flutter.io/flutter/material/Tween-class.html)
+
+This class represents a changing value between a range with a
+<code>[begin](http://docs.flutter.io/flutter/material/Tween/begin.html)</code> and
+<code>[end](http://docs.flutter.io/flutter/material/Tween/end.html)</code> value. To get the
+interpolated value for a given animation, call the
+<code>[evaluate()](http://docs.flutter.io/flutter/animation/Tween/evaluate.html)</code> method
+on the <code>[Tween](http://docs.flutter.io/flutter/material/Tween-class.html)</code> object. You
+can override the <code>[lerp()](http://docs.flutter.io/flutter/material/Tween/lerp.html)</code>
+(or <em>linear interpolation</em>) method for the
+<code>[Tween](http://docs.flutter.io/flutter/material/Tween-class.html)</code> object, which
+returns a value based on a given animation clock time.
+
+Use the subclasses of <code>[Tween](http://docs.flutter.io/flutter/material/Tween-class.html)</code>
```

Comment:
You are giving a list to memorize without the reader having seen anything that works yet. How is the reader expected to have any context to understand what you are describing? The behavior of a Tween makes no sense at all unless you see the pieces put together (This is based on experience interviewing other developers to write the original document. This isn't simply my opinion.)